packages/dashboards/app/styles/navi-dashboards/common.scss

```
@@ -7,21 +7,14 @@
 .navi-dashboard-container {
   .error-container,
   .loader-container {
-    .display-flex;
-    background-color: @denali-grey-300;
+    @extend .flex;
```

Any reason not just do this? In the past we used a mixin to do vendor prefixes (pre autoprefixer).
```suggestion
    display: flex;
```